README.md

```diff
@@ -47,23 +47,102 @@ After applying all the matchers, the output HIP source is produced.
-| [4.0.0](http://releases.llvm.org/download.html#4.0.0)   , [4.0.1](http://releases.llvm.org/download.html#4.0.1)    | [8.0](https://developer.nvidia.com/cuda-80-ga2-download-archive)         | +           | +         |
-| [5.0.0](http://releases.llvm.org/download.html#5.0.0)   , [5.0.1](http://releases.llvm.org/download.html#5.0.1)    , [5.0.2](http://releases.llvm.org/download.html#5.0.2) | [8.0](https://developer.nvidia.com/cuda-80-ga2-download-archive)         | +           | +         |
-| [6.0.0](http://releases.llvm.org/download.html#6.0.0)   , [6.0.1](http://releases.llvm.org/download.html#6.0.1)    | [9.0](https://developer.nvidia.com/cuda-90-download-archive)             | +           | +         |
-| [7.0.0](http://releases.llvm.org/download.html#7.0.0)   , [7.0.1](http://releases.llvm.org/download.html#7.0.1)    , [7.1.0](http://releases.llvm.org/download.html#7.1.0) | [9.2](https://developer.nvidia.com/cuda-92-download-archive)             | works only with the patch <br> due to the clang's bug [38811](https://bugs.llvm.org/show_bug.cgi?id=38811) <br> [patch for 7.0.0](patches/patch_for_clang_7.0.0_bug_38811.zip)\*\* <br> [patch for 7.0.1](patches/patch_for_clang_7.0.1_bug_38811.zip)\*\* <br> [patch for 7.1.0](patches/patch_for_clang_7.1.0_bug_38811.zip)\*\* | - <br> not working due to <br> the clang's bug [36384](https://bugs.llvm.org/show_bug.cgi?id=36384) |
-| [8.0.0](http://releases.llvm.org/download.html#8.0.0)   , [8.0.1](http://releases.llvm.org/download.html#8.0.1)    | [10.0](https://developer.nvidia.com/cuda-10.0-download-archive)          | works only with the patch <br> due to the clang's bug [38811](https://bugs.llvm.org/show_bug.cgi?id=38811) <br>[patch for 8.0.0](patches/patch_for_clang_8.0.0_bug_38811.zip)\*\* <br>[patch for 8.0.1](patches/patch_for_clang_8.0.1_bug_38811.zip)\*\*<br>| + |
-| [9.0.0](http://releases.llvm.org/download.html#9.0.0)   , [9.0.1](http://releases.llvm.org/download.html#9.0.1)    | [10.1](https://developer.nvidia.com/cuda-10.1-download-archive-base)     | +           | +         |
-| [10.0.0](http://releases.llvm.org/download.html#10.0.0) , [10.0.1](https://releases.llvm.org/download.html#10.0.1) | [**11.0**](https://developer.nvidia.com/cuda-11.0-download-archive), [11.0 Update 1](https://developer.nvidia.com/cuda-11.0-update1-download-archive)\*\*\*, [11.1](https://developer.nvidia.com/cuda-downloads)\*\*\* | +           | +         |
-| [**11.0.0**](https://releases.llvm.org/download.html#11.0.0) | [**11.0**](https://developer.nvidia.com/cuda-11.0-download-archive), [11.0 Update 1](https://developer.nvidia.com/cuda-11.0-update1-download-archive)\*\*\*, [11.1](https://developer.nvidia.com/cuda-downloads)\*\*\* | + <br/> **LATEST STABLE RELEASE** | + <br/> **LATEST STABLE RELEASE** |
+<table align="center">
```

Comment:
It used to be md format but now becomes html. Is this intended?

Reply:
It is intended, cause I was unable to apply HTML's `colspan` and `rowspan` analogues in GitHub's Markdown format (`||` doesn't work). Nonetheless, GitHub correctly parses Markdown documents with simple HTML injections.
Before (mostly in md format):
![before](https://user-images.githubusercontent.com/10241941/96709793-d93a3680-13a3-11eb-90b5-b5b1c8657210.png)
After (with HTML table injection)
![after](https://user-images.githubusercontent.com/10241941/96709482-69c44700-13a3-11eb-991f-e9fea0857c73.png)